### DISCUSSION TITLE
Advance blender user, want to get involved

### DISCUSSION THREAD

**Discussion by HNesher** [GimelStudio/GimelStudio#197] [General]
hey, my name is Hadar, I'm a blender user for around 5 - 6 years (mostly in shader/geometry nodes) and I love the idea of this project!
I wanted a node based image editor for quite a while.
I'm just now starting to learn programming but I am familiar with image processing algorithms.

Is there any way I could help or contribute to the project?

here are some projects i have done with shader/geo nodes:
[lego generator](https://www.instagram.com/p/CqP7zW6KR5A/?utm_source=ig_web_copy_link&igsh=MzRlODBiNWFlZA==)
[vector displacement revolver](https://www.instagram.com/p/C2W3dGEqF_i/?utm_source=ig_web_copy_link&igsh=MzRlODBiNWFlZA==)
[relax points](https://www.instagram.com/reel/C4D7yisiEJR/?utm_source=ig_web_copy_link&igsh=MzRlODBiNWFlZA==l)

**Comment by Correct-Syntax** [comment]
Hello @HNesher, thanks for wanting to help out. 

You should know that currently, the project is on a temporary hiatus since @iwoithe and I have both been quite busy since mid last year. We haven't yet picked up where we left off in planning.

I took a look at your project links. Nice stuff there. 👍 

To answer your question, maybe you could take a look at our ideas project board (I just invited you) and share some thoughts and feedback on our plans.  EDIT: here's the link https://github.com/GimelStudio/ideas/projects/1

We have had it in mind to rewrite the code and tailor the application towards different use-cases around the idea of node-editing for 2D graphics.

On the technical side, we're likely going to use QML (Qt) via [PySide](https://doc.qt.io/qtforpython-6/) for the interface and [Python](https://python.org) as the main language. We'll also continue to use GLSL and other python libraries for image processing. 

*However, this is all still subject to change since this is just where we left off.*  

If you'd like to work on the image processing that would be a great help. Feel free to suggest anything you think in that area.

What is your main interest and (future) use-case for Gimel Studio?
